test/unit/renderer/viz/expressions/binary.test.js

```diff
@@ -8,46 +8,46 @@ s.FALSE.toString = () => 's.FALSE';
 describe('src/renderer/viz/expressions/binary', () => {
     describe('error control', () => {
         describe('Signature NUMBERS_TO_NUMBER | NUMBER_AND_COLOR_TO_COLOR | COLORS_TO_COLOR', () => {
-            validateDynamicTypeErrors('mul', ['number', 'category']);
-            validateDynamicTypeErrors('mul', ['category', 'number']);
-            validateDynamicTypeErrors('mul', ['category', 'category']);
+            validateCompileTypeError('mul', ['number', 'category']);
```

Comment:
Because of the removal of the extra check on `binary.js`, these tests checked the existence of pseudo `checkLooseType` checks, but I removed those, see Jesus's comment